css/styles.css

```diff
@@ -56,27 +68,32 @@ section {
 
 - We want a border to the left and the right of the title only. The border should be 3px solid with the color "gold".)
 - There should be some space between the characters and the border.
-- The right border should end with the last character of the title. Hint: If you don't know how to do this, think about the display property.
+?? - The right border should end with the last character of the title. Hint: If you don't know how to do this, think about the display property.
```

Comment:
don't know what to do here.. maybe someone can help?

Reply:
Try this:
`display: inline;`
[display property](https://developer.mozilla.org/en-US/docs/web/css/display)
💪

Reply:
thx! fixed it now.